subsys/bluetooth/host/mesh/net.c

```diff
@@ -1167,7 +1167,7 @@ void bt_mesh_net_recv(struct net_buf_simple *data, s8_t rssi,
 		      enum bt_mesh_net_if net_if)
 {
 	struct net_buf_simple *buf = NET_BUF_SIMPLE(29);
-	struct bt_mesh_net_rx rx;
+	struct bt_mesh_net_rx rx = { .rssi = rssi };
```

Comment:
I guess the commit should say Fix not including...

Reply:
Well, it depends how you read it. If you read it as "Fix <feature>" and the <feature> is "including RSSI in the RX context" then the current commit message makes sense.

Reply:
Right maybe a description then, anyway other than that is looks fine.

Reply:
I changed it to the form that you were expecting, and added an extra sentence to the commit message body as well.